buf.gen.yaml

```
@@ -1,6 +1,6 @@
 version: v2
 clean: true
 plugins:
-  - remote: buf.build/protocolbuffers/go
+  - remote: buf.build/protocolbuffers/go:v1.36.0
```

Could we use the latest version?
```suggestion
  - remote: buf.build/protocolbuffers/go:v1.36.3
```